src/main/kotlin/com/terraformation/backend/customer/db/ProjectStore.kt

```diff
@@ -132,38 +129,45 @@ class ProjectStore(
       with(PROJECT_INTERNAL_USERS) {
         dslContext
             .insertInto(this)
-            .set(PROJECT_ID, projectId)
-            .set(USER_ID, userId)
-            .set(PROJECT_INTERNAL_ROLE_ID, role)
-            .set(ROLE_NAME, roleName)
+            .columns(PROJECT_ID, USER_ID, PROJECT_INTERNAL_ROLE_ID, ROLE_NAME)
```

Comment:
FYI, you can add the columns as additional arguments to `insertInto` rather than calling `.columns`.